### PR TITLE
use standardSourceTestFile by default in python templates

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Update templates to use the file based standard tests introduced in #1333

### CODE REVIEW

File: buildSrc/src/main/groovy/airbyte-standard-source-test-file.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ class AirbyteStandardSourceTestFilePlugin implements Plugin<Project> {
     void apply(Project project) {
         def config = project.extensions.create('airbyteStandardSourceTestFile', AirbyteStandardSourceTestFileConfiguration)
 
-        project.task('standardFsSourceTest') {
+        project.task('standardSourceTestFile') {
```
Comment (sherifnada) [inline buildSrc/src/main/groovy/airbyte-standard-source-test-file.gradle:27]: missed making this rename in #1333

File: airbyte-integrations/connector-templates/source-python/README.md.hbs [hunk 2]
```diff
@@ -57,7 +57,7 @@ docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-{{dashCase name
 
 ### Integration Tests
 1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-{{dashCase name}}:standardSourceTestPython` to run the standard integration test suite.
-1. To run additional integration tests, place your integration tests in the `integration_tests` directory and run them with `pytest integration_tests`.
+1. To run additional integration tests, place your integration tests in a new directory `integration_tests` and run them with `pytest integration_tests`.
```
Comment (michel-tricot) [inline airbyte-integrations/connector-templates/source-python/README.md.hbs:60]: Should we have a gradle rule instead? (and explain that it runs pytest behind the scene?

Does it mean that the tests don't run in docker?

File: airbyte-integrations/connector-templates/source-singer/README.md.hbs [hunk 3]
```diff
@@ -54,8 +54,8 @@ docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-{{dashCase name
 ```
 
 ### Integration Tests
-1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-{{dashCase name}}-singer:standardSourceTestPython` to run the standard integration test suite.
-1. To run additional integration tests, place your integration tests in the `integration_tests` directory and run them with `pytest integration_tests`.
+1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-{{dashCase name}}-singer:integrationTest` to run the standard integration test suite.
+1. To run additional integration tests, create a directory `integration_tests` which contain your tests and run them with `pytest integration_tests`.
```
Comment (michel-tricot) [inline airbyte-integrations/connector-templates/source-singer/README.md.hbs:58]: same as above

**Review by michel-tricot — APPROVED** [review]

File: airbyte-integrations/connector-templates/source-python/README.md.hbs (hunk 2 above)
Comment (cgardens) [reply airbyte-integrations/connector-templates/source-python/README.md.hbs:60]: my understanding of how we are handling non-standard test integration tests is poor. i think we need to document a bit more what that environment looks like. how do you add deps to it (is it same deps as test)? etc. fine with it not being in this PR as it's a bit of a scope creep. but if it is indeed under-documented can you create an issue to document what we need to here.

**Review by cgardens — APPROVED** [review]

File: airbyte-integrations/connector-templates/source-python/README.md.hbs (hunk 2 above)
Comment (sherifnada) [reply airbyte-integrations/connector-templates/source-python/README.md.hbs:60]: I like the idea of a gradle rule, such that if they implement things in a particular it works automatically with `integrationTest` task

Comment (sherifnada) [reply airbyte-integrations/connector-templates/source-python/README.md.hbs:60]: created issue: https://github.com/airbytehq/airbyte/issues/1371

**Pull request merged**